ccd-definition/FixedLists/CMOActionStatus.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "CMOActionStatus",
+    "ListElementCode": "SEND_TO_ALL_PARTIES",
+    "ListElement": "Yes, it can be sealed and sent to parties",
+    "DisplayOrder": 1
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "CMOActionStatus",
+    "ListElementCode": "LOCAL_AUTHORITY_CHANGES",
```

Comment:
Bit vague name. What is important here IMO is that LA was **requested** to make changes.

Reply:
that's correct @geek-soft, we could make it **_JUDGE_REQUESTS_LOCAL_AUTHORITY_CHANGES_**, given this will be requested by the Judge?